src/components/Heading/Heading.stories.tsx

```diff
@@ -52,10 +53,47 @@ export const Heading1AsHeading4: StoryObj<Args> = {
   },
 };
 
-export const Heading4ColorNeutral: StoryObj<Args> = {
-  args: {
-    children: "Neutral color Heading 4",
-    color: "neutral",
-    size: "h4",
+/**
+ * 1) Insufficient color contrast is expected since we are displaying all color variants on various color backgrounds.
+ */
+export const Variants: StoryObj<Args> = {
+  ...Heading1,
+  render: (args) => {
+    const headings: React.ReactNode[] = [];
+    VARIANTS.forEach((variant) => {
+      if (variant !== "info")
+        headings.push(
+          <Heading {...args} key={`${variant}`} variant={variant}>
+            {variant}
+          </Heading>,
+        );
+    });
+    return (
+      <div className={styles["variant__table"]}>
+        <div className={styles["variant--white"]}>{headings}</div>
+        <div className={styles["variant--light"]}>{headings}</div>
+        <div className={styles["variant--dark"]}>{headings}</div>
+      </div>
+    );
+  },
+  parameters: {
+    axe: {
+      disabledRules: ["color-contrast"] /* 1 */,
+    },
+  },
+};
+
+export const Sizes: StoryObj<Args> = {
+  render: () => {
+    const sizes = ["h1", "h2", "h3", "h4", "h5", "h6"] as const;
+    const combinations = sizes.map((as: typeof sizes[number]) => {
+      const headings = sizes.map((size: typeof sizes[number]) => (
```

Comment:
super nit: are the type annotations in `map`? I'd expect TS to be able to infer it. If not, maybe we can use the `HeadingElement` type instead (more readable)

Reply:
Yea think we can, thanks for the catch